### PR TITLE
VE-2045[4]: add metrics collecting to the Java wrapper

### PR DESCRIPTION
**Pull request by Toktar**

### CODE REVIEW

File: wrappers/java/src/main/java/org/hyperledger/indy/sdk/metrics/Metrics.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    /*
+     * STATIC CALLBACKS
+     */
+
+    /**
+     * Callback used when method with string result completes
+     */
+    private static Callback stringCompleteCb = new Callback() {
+
+        @SuppressWarnings({"unused", "unchecked"})
```
Comment (skhoroshavin) [inline wrappers/java/src/main/java/org/hyperledger/indy/sdk/metrics/Metrics.java:24]: Why do we need these?

File: wrappers/java/src/test/java/org/hyperledger/indy/sdk/metrics/CollectMetricsTest.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import org.junit.Test;
+import static org.junit.Assert.assertNotNull;
+
+
+public class CollectMetricsTest extends IndyIntegrationTest {
+
+	@Test
+	public void testCollectMetricsMethod() throws Exception {
+		String metrics_map = Metrics.collectMetrics().get();
+		assertNotNull(metrics_map);
```
Comment (skhoroshavin) [inline wrappers/java/src/test/java/org/hyperledger/indy/sdk/metrics/CollectMetricsTest.java:13]: I would also suggest to test that returned string is a valid JSON

File: wrappers/java/src/main/java/org/hyperledger/indy/sdk/metrics/Metrics.java (hunk 1 above)
Comment (Toktar) [reply wrappers/java/src/main/java/org/hyperledger/indy/sdk/metrics/Metrics.java:24]: This annotation is used for all callbacks in libindy. I suggest leaving it.

File: wrappers/java/src/test/java/org/hyperledger/indy/sdk/metrics/CollectMetricsTest.java (hunk 2 above)
Comment (Toktar) [reply wrappers/java/src/test/java/org/hyperledger/indy/sdk/metrics/CollectMetricsTest.java]: Added a few checks for a result JSON.

**Review by skhoroshavin — APPROVED** [review]

**Pull request merged**